app/src/cnm/java/org/eyeseetea/malariacare/data/sync/importer/strategies/PullControllerStrategy.java

```diff
@@ -49,7 +64,7 @@ public void pull(PullFilters pullFilters, IPullController.Callback callback, Con
                     throw new AutoconfigureException();
                 }
             }
-            if (organisationUnit != null || pullFilters.isDemo()) {
+            if (pullFilters.isDemo()) {
                 callback.onComplete();
             } else {
                 mPullController.pullMetada(pullFilters, callback);
```

Comment:
when the pull is executed in autoconfigure mode, this method is executed but without parameters in pullFilters to download metadata